src/main/java/uk/gov/hmcts/reform/waworkflowapi/controllers/startworkflow/StartWorkflowController.java

```diff
@@ -5,26 +5,43 @@
 import io.swagger.annotations.ApiResponses;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.waworkflowapi.external.taskservice.Task;
+import uk.gov.hmcts.reform.waworkflowapi.external.taskservice.TaskManagerService;
 
+import java.util.Optional;
+
+import static org.springframework.http.ResponseEntity.created;
 import static org.springframework.http.ResponseEntity.noContent;
 
 @RestController
 public class StartWorkflowController {
 
     private final Logger log = LoggerFactory.getLogger(StartWorkflowController.class);
 
+    private final TaskManagerService taskManagerService;
+
+    @Autowired
```

Comment:
Don't really mind `@Autowired` but you can avoid using them e.g. https://github.com/hmcts/ia-case-api/blob/master/src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/AdvancedFinalBundlingStitchingCallbackHandler.java#L37-L45

Reply:
I kind of feel if you are going to use magic at least give a pointer to how that magic is working. If you could get by with no other spring annotations I could see then the class would be cleaner but you have to add the @RestController on there so it is already polluted.

Reply:
Yes it's true in earlier versions you had to.  Let's discuss further in the dev WoW session.